config/settings.yml

```diff
@@ -3,6 +3,7 @@ Default: &defaults
   port: 3000
   approot: "http://localhost:3000"
   copyright: Insert copyright statement here
+  allowAuthDummy: True # Uncomment to allow Auth Dummy login
```

Comment:
It actually should have been commented out (I believe we can only pass `Text` values here, not real `Bool`).

Seems I was a bit too fast on the "needs review" button, just pushed a bunch of indentation fixes to satisfy `stack test`'s parser. I'll wait for travis to return green..